0-setup-libraries.ipynb

```diff
@@ -101,53 +2922,10 @@
-      "Requirement already satisfied: torch_scatter in /home/cms.woodson/.local/lib/python2.7/site-packages (1.2.0)\n",
-      "Requirement already satisfied: torch_sparse in /home/cms.woodson/.local/lib/python2.7/site-packages (0.4.0)\n",
-      "Requirement already satisfied: torch_cluster in /home/cms.woodson/.local/lib/python2.7/site-packages (1.3.0)\n",
-      "Requirement already satisfied: pandas==0.24.0 in /usr/lib64/python2.7/site-packages (0.24.0)\n",
-      "Requirement already satisfied: future in /home/cms.woodson/.local/lib/python2.7/site-packages (from torch) (0.17.1)\n",
-      "Requirement already satisfied: numpy in /home/cms.woodson/.local/lib/python2.7/site-packages (from torch) (1.16.4)\n",
-      "Requirement already satisfied: six in /home/cms.woodson/.local/lib/python2.7/site-packages (from torchvision) (1.12.0)\n",
-      "Requirement already satisfied: pillow>=4.1.1 in /home/cms.woodson/.local/lib/python2.7/site-packages (from torchvision) (6.0.0)\n",
-      "Requirement already satisfied: scikit-learn>=0.19.1 in /home/cms.woodson/.local/lib/python2.7/site-packages (from scikit-optimize) (0.20.3)\n",
-      "Requirement already satisfied: scipy>=0.14.0 in /usr/lib64/python2.7/site-packages (from scikit-optimize) (1.2.0)\n",
-      "Requirement already satisfied: networkx in /home/cms.woodson/.local/lib/python2.7/site-packages (from torch_geometric) (2.2)\n",
-      "Requirement already satisfied: plyfile in /home/cms.woodson/.local/lib/python2.7/site-packages (from torch_geometric) (0.7)\n",
-      "Requirement already satisfied: rdflib in /home/cms.woodson/.local/lib/python2.7/site-packages (from torch_geometric) (4.2.2)\n",
-      "Requirement already satisfied: h5py in /home/cms.woodson/.local/lib/python2.7/site-packages (from torch_geometric) (2.9.0)\n",
-      "Requirement already satisfied: pytz>=2011k in /usr/lib/python2.7/site-packages (from pandas==0.24.0) (2018.9)\n",
-      "Requirement already satisfied: python-dateutil>=2.5.0 in /usr/lib/python2.7/site-packages (from pandas==0.24.0) (2.7.5)\n",
-      "Requirement already satisfied: decorator>=4.3.0 in /home/cms.woodson/.local/lib/python2.7/site-packages (from networkx->torch_geometric) (4.4.0)\n",
-      "Requirement already satisfied: isodate in /home/cms.woodson/.local/lib/python2.7/site-packages (from rdflib->torch_geometric) (0.6.0)\n",
-      "Requirement already satisfied: pyparsing in /usr/lib/python2.7/site-packages (from rdflib->torch_geometric) (2.3.1)\n",
-      "\u001b[33mYou are using pip version 19.0.1, however version 19.1.1 is available.\n",
-      "You should consider upgrading via the 'pip install --upgrade pip' command.\u001b[0m\n"
-     ]
-    }
-   ],
-   "source": [
-    "import sys\n",
-    "!{sys.executable} -m pip install torch torchvision scikit-optimize --user"
+    "The new kernel you just made will then show up in the various Jupyter after logging out and back in (restarting your server)."
```

Comment:
Won't it also show up if you just refresh the page? That's what I've been doing an it seems to work for me. Might confuse the student less if they just have to refresh.

Reply:
I think you're right. Fixed.